components/contact-list.tsx

```diff
@@ -4,6 +4,7 @@ import { ContactListSkeleton } from "~/components/ui/skeleton-loading";
 import { anchorTransformer } from "~/lib/htmr-transformers";
 import { Contact } from "~/lib/provinces";
 import { isNotEmpty, stripTags } from "~/lib/string-utils";
+import { getContactMetaTitle } from "~/pages/provinces/[provinceSlug]/[contactSlug]";
 
```

Comment:
This is weird, the components require the pages.

Reply:
I think, we can create utilities under `~lib/meta`
Then move to the `getMetaProvince` and the `getMetaContactDetail` maybe 🤔